### DISCUSSION TITLE
assign default customer

### DISCUSSION THREAD

**Discussion by aloistr** [kimai/kimai#3650] [Questions & Answers]
We use kimai for our software development team within a single company, which is our only "customer".
can we get default assignment to this customer, so that we do not always have to select it when a work time is entered?

---------
question 2: It would also be good if each user could have a default project assigned (which he can change himself) so
that he does not neen to select it, unless he wants some work time specifically assigned to a different project.

**Comment by kevinpapst** [comment]
> can we get default assignment to this customer, so that we do not always have to select it when a work time is entered?

You never have to select a customer

For the other one: use "restart" or the calendar

**Reply by aloistr** [reply]
Sorry, I do not know what you mean by 'use restart', nor do I understand in what sense the use of the calendar solves the problem.
Of course I am very new to your software. We will switch to a paid version for 5 users once we find out how to use it.

**Reply by kevinpapst** [reply]
If you are seeking support for the Cloud version, then this here is the wrong place. 

Please use https://www.kimai.cloud/de/support-center/request instead (german possible)

**Reply by aloistr** [reply]
mein Kollege hat die Anmeldung für die Cload gemacht, aber ich bin auch Admin im Cloudkonto.
ich kommem aber mit meiner id nicht beim cloud support center rein.
Mit seiner id komne ich schon rein, aber ich will nich tunter der Identität meines Kollegen dort auftreten.
gelöst: komme mit meiner Github id rein.
